### PR TITLE
Skeleton Tailwind Plugin Update

### LINKED ISSUE #650: Update Plugin to support Tailwind Intellisense
### Current Behavior

I've heard reports it works for Daisy UI, and am sure it works locally in the Skeleton project, so we'll need to see how to carry this through to end users.

### Steps To Reproduce

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by endigo9740**
Fixes #650

This will not only improve and flush out our Tailwind plugin, but also extend it to auto-generate most if not all design token classes. This will provide a number of benefits:

1. Easier to maintain, since most are generate in a loop
2. Reduction of human error - no missing a color variation by accident
3. Introduce Intellisense suggestions for these classes

This may take a couple days to complete, but feedback is welcome as I progress.

### CODE REVIEW

**Comment by endigo9740** [comment]
@ryceg

File: src/lib/tailwind/tokens/border-radius.cjs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// Design Tokens: Border Radius
+// Doc: https://www.skeleton.dev/elements/tokens
+
+// const settings = require('../settings.cjs');
+
+module.exports = () => {
+    return {
+        // Base
+        '.rounded-token': { 'border-radius': 'var(--theme-rounded-base)' },
+        '.rounded-tl-token': { 'bordertop-left-radius': 'var(--theme-rounded-base)' },
```
Comment (ryceg) [inline src/lib/tailwind/tokens/border-radius.cjs:10]: typo?

File: src/lib/tailwind/tokens/rings.cjs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    '--tw-ring-inset': 'inset'
+};
+
+module.exports = () => {
+    const classes = {
+        // Ring Outline (for cards)
+        // Example: .ring-outline-token
+        '.ring-outline-token': {
+            ...ringOutlineShared,
+            '--tw-ring-color': 'rgb(23 23 23 / 0.05);', // neutral-900
```
Comment (ryceg) [inline src/lib/tailwind/tokens/rings.cjs:22]: can we extract the value to a `const` and name it so we know what 23 23 23 / 0.05 is?

File: src/lib/tailwind/tokens/border-radius.cjs (hunk 1 above)
Comment (endigo9740) [reply src/lib/tailwind/tokens/border-radius.cjs:10]: Yep - missed the dash!

**Comment by endigo9740** [comment]
@ryceg I've implemented one of the items mentioned above and commented on the other. For now I'm going to aim to merge this into `dev`. I welcome anyone continue testing up until release though.

**Pull request merged**